# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/Overscroll.h
# Commit: 76279f0a6d33
# Full Hash: 76279f0a6d33d6fe128a9f3aabe5393e52181818
# Author: James Willcox <snorp@snorp.net>
# Date: 2020-06-08 21:38:11
# Regressor Bug: 1536833
# File Overlap Count: 1
# Description:
#   Bug 1536833 - Fix overscroll effect on Android r=botond,geckoview-reviewers,agi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78130
# ==============================================================================

diff -r ad757ab87752 -r 76279f0a6d33 gfx/layers/apz/src/Overscroll.h
--- a/gfx/layers/apz/src/Overscroll.h	Sun Jun 07 23:15:45 2020 +0000
+++ b/gfx/layers/apz/src/Overscroll.h	Mon Jun 08 17:13:28 2020 +0000
@@ -112,7 +112,8 @@
     RefPtr<GeckoContentController> controller =
         mApzc.GetGeckoContentController();
     if (controller && (aShouldOverscrollX || aShouldOverscrollY)) {
-      controller->UpdateOverscrollOffset(aOverscroll.x, aOverscroll.y,
+      controller->UpdateOverscrollOffset(mApzc.GetGuid(),
+                                         aOverscroll.x, aOverscroll.y,
                                          mApzc.IsRootContent());
       aOverscroll = ParentLayerPoint();
     }
@@ -122,7 +123,8 @@
     RefPtr<GeckoContentController> controller =
         mApzc.GetGeckoContentController();
     if (controller) {
-      controller->UpdateOverscrollVelocity(aVelocity.x, aVelocity.y,
+      controller->UpdateOverscrollVelocity(mApzc.GetGuid(),
+                                           aVelocity.x, aVelocity.y,
                                            mApzc.IsRootContent());
     }
   }